code/game/machinery/computer/groundside_operations.dm

```diff
@@ -235,7 +235,7 @@
 			log_announcement("[key_name(usr)] has announced the following: [input]")
 
 		if("award")
-			print_medal(usr, src)
+			open_medal_panel(usr, src)
```

Comment:
Note: As far as I can tell this is impossible to access atm because the interface no longer has "award" as an option in its `ui_interact`.

Reply:
see above